### ISSUE TITLE
"No such group or event: FileExplorer *" when g:nvim_tree_disable_netrw = 0

### ISSUE THREAD

**Issue by dutchgecko** [kyazdani42/nvim-tree.lua#254]
I have the following vim-tree config:

```vim
let g:nvim_tree_ignore = [
    \ '.git',
    \ 'node_modules',
    \ '.cache',
    \ '__pycache__',
\ ]
let g:nvim_tree_follow = 1
let g:nvim_tree_indent_markers = 1
let g:nvim_tree_disable_netrw = 0
```

When opening nvim I'm presented with this error:

```
Error detected while processing /home/gecko/.config/nvim/plugged/nvim-tree.lua/plugin/tree.vim:
line   13:
E216: No such group or event: FileExplorer *
```

Setting `g:nvim_tree_disable_netrw` to 1 (the default) stops the error from occurring.

**Comment by CuriousSyrup** [comment]
Same for me:
let g:nvim_tree_disable_netrw = 0
generates the same error.

**Comment by kyazdani42** [comment]
I did not manage to reproduce but i can disable this error by silenting the autocmd. I'd like to understand where this FileExplorer autocmd comes from though, it was added by an user and i'm not quite sure what it does ^^'

**Comment by kyazdani42** [comment]
should be fixed in e5bd242
